study.md

```diff
@@ -22,7 +22,8 @@ In your own words, explain one reason why we use "this". In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+this appears to have many applications. That said, we often use this as to stand by the
+principal: DRY. it keeps us from needing to write multiple functions or conditional statements.
```

Comment:
Good and the way we're using `this` so far is that when an object has a method, using `this` inside the method will refer to the object the method is being called on.